modules/govuk_jenkins/templates/jobs/deploy_app_downstream.yaml.erb

```diff
@@ -0,0 +1,64 @@
+---
+- job:
+    name: Deploy_App_Downstream
+    display-name: Deploy_App_Downstream
+    project-type: freestyle
+    description: "Kicks off an appliction deploy in a downstream environment"
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+          artifact-num-to-keep: 5
+    builders:
+      - shell: |
+          # Workaround for our inconsistent repo vs. deployment naming
+          case "$TARGET_APPLICATION" in
+          <% @applications.keys.each do |app| %>
+            <%= app %>)
```

Comment:
This is necessary syntax for [the case statement](https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_03.html). On a slight tangent: since your team is working on Smart Answers, do you think there's any scope to remove the extremely annoying inconsistency between the repo name and the infrastructure name for Smart Answers? This repo is [basically the only reason why we need this logic](https://github.com/alphagov/govuk-puppet/blob/master/hieradata_aws/common.yaml#L145).